### PR TITLE
Added contentful-scheduler gem

### PR DESCRIPTION
**Pull request by pedropbr**

### CODE REVIEW

File: docs/tools/others.md [hunk 1]
```diff
@@ -4,9 +4,14 @@ page: :docsToolsOthers
 
 ## Contentful Webhook Listener
 
-A Simple HTTP Webserver with pluggable behavior for listening to API Webhooks<br>
+A Simple HTTP Webserver with pluggable behavior for listening to Webhooks API.<br>
```
Comment (realityking) [inline docs/tools/others.md]: I think just removing `API` from this sentence would make it clearer.

**Comment by pedropbr** [comment]
@realityking true, done

File: docs/tools/others.md [hunk 2]
```diff
@@ -16,6 +21,7 @@ This tutorial will show you how you can use two new Amazon services, Lambda and
 
 In this article, we will expose how to manage the publishing time of content.<br>
 [Read the Tutorial](/developers/docs/tutorials/general/scheduling-posts/)
```
Comment (realityking) [inline docs/tools/others.md]: Missing line break.

**Comment by pedropbr** [comment]
@realityking where? can you add it please?
there's nothing work with the UI
https://github.com/contentful/slash-developers/blob/pedropbr-patch-1/docs/tools/others.md

**Comment by pedropbr** [comment]
@realityking done

**Comment by realityking** [comment]
👌

**Pull request merged**